modules/nodes-scheduler-taints-tolerations-about.adoc

```diff
@@ -142,70 +142,71 @@ This feature, in combination with `tolerationSeconds`, allows a pod to specify h
+ 
+The scheduler checks for these taints on nodes before scheduling pods. If the taint is present, the pod is scheduled on a different node. Because the scheduler checks for taints and not the actual Node conditions, you configure the scheduler to ignore some of these node conditions by adding appropriate Pod tolerations. 
 
-{product-title} has TaintNodesByCondition feature enabled by default, so node lifecycle controller automatically creates taints corresponding to Node conditions like memory pressure, disk pressure. Similarly the scheduler does not check Node conditions; instead the scheduler checks taints. This assures that Node conditions don’t affect what’s scheduled onto the Node. The user can choose to ignore some of the Node’s problems (represented as Node conditions) by adding appropriate Pod tolerations. Note that TaintNodesByCondition only taints nodes with NoSchedule effect.
+The DaemonSet controller automatically adds the following tolerations to all daemons, to prevent DaemonSets from breaking. 
 
-The DaemonSet controller automatically adds the following NoSchedule tolerations to all daemons, to prevent DaemonSets from breaking.
-
-    node.kubernetes.io/memory-pressure
-    node.kubernetes.io/disk-pressure
-    node.kubernetes.io/out-of-disk (only for critical pods)
-    node.kubernetes.io/unschedulable (1.10 or later)
-    node.kubernetes.io/network-unavailable (host network only)
-
-Adding these tolerations ensures backward compatibility. You can also add arbitrary tolerations to DaemonSets
+* node.kubernetes.io/memory-pressure
+* node.kubernetes.io/disk-pressure
+* node.kubernetes.io/out-of-disk (only for critical pods)
+* node.kubernetes.io/unschedulable (1.10 or later)
+* node.kubernetes.io/network-unavailable (host network only)
 
+Adding these tolerations ensures backward compatibility. You can also add arbitrary tolerations to DaemonSets.
 
 [id="nodes-scheduler-taints-tolerations-about-taintBasedEvictions-{context}"]
-== Taint Based Evictions
+== Understanding evicting pods by condition (Taint-Based Evictions)
 
-Earlier we mentioned the NoExecute taint effect, which affects pods that are already running on the node as follows
+The Taint-Based Evictions feature, enabled by default, evicts pods from a node that experiences specific conditions, such as `not-ready` and `unreachable`. 
+When a node experiences one of these conditions, {product-title} automatically adds taints to the node and starts evicting pods and reschedules the pods on different nodes.
 
-    pods that do not tolerate the taint are evicted immediately
-    pods that tolerate the taint without specifying tolerationSeconds in their toleration specification remain bound forever
-    pods that tolerate the taint with a specified tolerationSeconds remain bound for the specified amount of time
+The following conditions trigger taint-based evictions: 
```

Comment:
This statement is not true. Node-Controller won't taint nodes with `out-of-disk`, `memory-pressure`etc. Only `not-ready` and `unreachable` taints are applied as part of taint based evictions.

I copied this section from upstream docs so that users can know, what all taints are built into kubernetes.

Reply:
@ravisantoshgudimetla I removed this sentence and moved the taint list into the introductory section. Does this make sense?